### PR TITLE
Added Button in UI to link Zulip

### LINKED ISSUE #1038: Add button in UI to link Zulip
## Description
As a user, I need to add a button to link our Zulip communication platform, so that I can have smooth communication through project.

### Update [Required]
- [ ] Screen shot in PR to show Button added
- [ ] Link to organisation group, Zulip

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by yash2189**
# Description
To add button in UI to link Zulip
Fixes #1038  

# Type of Change:

- User Interface

# How Has This Been Tested?
Successfully added button in UI to link Zulip. When a registered user is logged in the button is shown on the home page.

Screenshot:

Button_ZULIP

# Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

**Review by sanchibansal340 — DISMISSED** [review]
Looks good to me

**Review by gaurivn — APPROVED** [review]
Great work, thank you

**Comment by keshakaneria** [comment]
Build is failing, did you try to edit anything after submitting the PR?
Details of this failure: This is a pull request build.
It is running a build against the merge commit, after merging #1091 Added Button in UI to link Zulip.
Any changes that have been made to the develop branch before the build ran are also included.

File: vms/home/templates/home/home.html [hunk 1]
```diff
@@ -7,23 +7,28 @@
...
+        <button type="button" class="btn btn-primary btn-lg">Connect With us on Zulip</button></a>
+    {% endif %}
+    {% if error %}
+    <div class="alert alert-danger" role="alert">
+        You will need to have atleast one organisation to Sign up as a volunteer
     </div>
-{% endblock %}
+    {% endif %}
+</div>
+{% endblock %}
```
Comment (gaurivn) [inline vms/home/templates/home/home.html:34]: Please add a newline at the end of this file - how it used to be before.

**Review by gaurivn — CHANGES_REQUESTED** [review]

**Comment by keshakaneria** [comment]
@yash2189 Push new commit with requested changes and squash them to single commit for final merge

**Comment by yash2189** [comment]
@keshakaneria I did not edit anything after submitting the PR

**Comment by keshakaneria** [comment]
> @keshakaneria I did not edit anything after submitting the PR

add the last line needed as suggested by @gaurivn locally and push the commit again

**Comment by yash2189** [comment]
Do I need to squash the commits after I push the new commit?

**Comment by keshakaneria** [comment]
> Do I need to squash the commits after I push the new commit?

Every new edit will have to be pushed which will eventually increase the commits. Thus you will have to squash them to turn it into single commit.

**Comment by yash2189** [comment]
> > Do I need to squash the commits after I push the new commit?
> 
> Every new edit will have to be pushed which will eventually increase the commits. Thus you will have to squash them to turn it into single commit.

I have done the required changes in the file will you please check @keshakaneria

File: vms/home/templates/home/home.html [hunk 2]
```diff
@@ -31,4 +31,4 @@ <h1 class="index">Volunteer Management System</h1>
     </div>
     {% endif %}
 </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```
Comment (keshakaneria) [inline vms/home/templates/home/home.html:34]: What did you change? It's same as before. You were supposed to add a new line as suggested.

**Review by keshakaneria — CHANGES_REQUESTED** [review]

File: vms/home/templates/home/home.html (hunk 2 above)
Comment (yash2189) [reply vms/home/templates/home/home.html:34]: I just need to add the `<br>` tag if I'm not wrong?

Comment (gaurivn) [reply vms/home/templates/home/home.html:34]: The warning red circle minus sign (in the previous commit) went, that was all that was needed, it was added, won't be evident in the commit, I guess. Like [this file](https://github.com/anitab-org/vms/blob/develop/vms/home/templates/home/home.html), we won't see in commit message, but on trying to edit, newline shows at the end.

Comment (keshakaneria) [reply vms/home/templates/home/home.html:34]: Not a <br> tag. Just an Enter. According to the code quality, one must leave an empty line at the end of the file.

Comment (yash2189) [reply vms/home/templates/home/home.html:34]: I did add a new line now but I'm unable to see it when I open the file.

**Comment by yash2189** [comment]
Can you please check again @gaurivn @keshakaneria

File: vms/home/templates/home/home.html (hunk 2 above)
Comment (gaurivn) [reply vms/home/templates/home/home.html:34]: As long as the warning, "No newline at end of file" is not there, it's good, it's already added and it's fine if you're unable to see.

**Comment by keshakaneria** [comment]
> Can you please check again @gaurivn @keshakaneria

Let the build finish, its almost perfect so no need to worry! Once the build is finished, squash them so that i can put a tag to finally get merged.

**Comment by yash2189** [comment]
@keshakaneria can you please tell me how do I squash these 3 commits since I made a mistake while doing the same previously .Also I am not acquainted with the rebase and squash commands in git.

**Comment by gaurivn** [comment]
Yay, it's working, and @yash2189, for squashing, do check [this](https://gist.github.com/longtimeago/f7055aa4c3bba8a62197) out.

**Comment by yash2189** [comment]
> Yay, it's working, and @yash2189, for squashing, do check [this](https://gist.github.com/longtimeago/f7055aa4c3bba8a62197) out.

Yes I did squash them a moment back, the build is running.

**Comment by keshakaneria** [comment]
> @gaurivn @keshakaneria can you please tell me how do I squash these 3 commits since I made a mistake while doing the same previously .Also I am not acquainted with the rebase and squash commands in git.

Write `git rebase --i HEAD~4` since you want to squash last 4 recent commits. Check it its the same which you want to squash. change `pick` to `s` to squash. Then type `:!qw` to save and go back. Push it again and your done with squashing.

**Comment by keshakaneria** [comment]
> > Yay, it's working, and @yash2189, for squashing, do check [this](https://gist.github.com/longtimeago/f7055aa4c3bba8a62197) out.
> 
> Yes I did squash them a moment back, the build is running.

Thats amazing! Congratulations for your first PR 🥳

**Review by keshakaneria — APPROVED** [review]

**Review by gaurivn — APPROVED** [review]

**Review by isabelcosta — APPROVED** [review]
Thank you for your contribution @yash2189 👏 🎉

**Pull request merged**